src/CorrectPlayerMovePredictionPacket.php

```diff
@@ -14,31 +14,34 @@
 
 namespace pocketmine\network\mcpe\protocol;
 
+use pocketmine\math\Vector2;
 use pocketmine\math\Vector3;
 use pocketmine\network\mcpe\protocol\serializer\PacketSerializer;
 
 class CorrectPlayerMovePredictionPacket extends DataPacket implements ClientboundPacket{
 	public const NETWORK_ID = ProtocolInfo::CORRECT_PLAYER_MOVE_PREDICTION_PACKET;
 
-	public const PREDICTION_TYPE_VEHICLE = 0;
-	public const PREDICTION_TYPE_PLAYER = 1;
+	public const PREDICTION_TYPE_PLAYER = 0;
+	public const PREDICTION_TYPE_VEHICLE = 1;
 
 	private Vector3 $position;
 	private Vector3 $delta;
 	private bool $onGround;
 	private int $tick;
 	private int $predictionType;
+	private ?Vector2 $vehicleRotation;
 
 	/**
 	 * @generate-create-func
 	 */
-	public static function create(Vector3 $position, Vector3 $delta, bool $onGround, int $tick, int $predictionType) : self{
+	public static function create(Vector3 $position, Vector3 $delta, bool $onGround, int $tick, int $predictionType, ?Vector2 $vehicleRotation) : self{
 		$result = new self;
 		$result->position = $position;
 		$result->delta = $delta;
 		$result->onGround = $onGround;
 		$result->tick = $tick;
 		$result->predictionType = $predictionType;
+		$result->vehicleRotation = $vehicleRotation;
```

Comment:
why is the check removed here?

Reply:
> why is the check removed here?

PHP-CS-Fixer complained about it, and whether its set is only relevant in the encoding process where it needs to be present